README.md

```diff
@@ -52,18 +61,25 @@ the value cannot be deciphered and an exception is thrown.
 
 ## Development
 Prerequisites:
-* configured `az` and `aws` CLI tools (run `az login` and `aws configure --profile YOUR_AWS_PROFILE_NAME`)
+* configured access to cloud providers
+  * installed Azure CLI `az` (and run `az login`) 
+  * installed AWS CLI `aws` (and run `aws configure --profile YOUR_AWS_PROFILE_NAME`)
+  * installed GCP CLI `gcloud` (and run `gcloud auth login` or `gcloud auth application-default login`)
 * installed `terraform` (https://www.terraform.io) and `jq` (https://stedolan.github.io/jq) to setup local env
 * installed `docker` and `docker-compose` to run & develop the app
 
-```
+```bash
+export NAME_PREFIX= # your name/nickname to make your resource unique & recognizable
+export AWS_PROFILE= # your AWS profile name e.g. Keboola-Dev-Platform-Services-AWSAdministratorAccess
+
 cat <<EOF > ./provisioning/local/terraform.tfvars
-name_prefix = "name" # your name/nickname to make your resource unique & recognizable
+name_prefix = "${NAME_PREFIX}"
+gcp_project = "kbc-dev-platform-services"

```

Comment:
Proc je tu `gcp_project` jako variable? AWS account a AZ subscription mame taky zapeceny v TF.

Reply:
protože je to potřeba pro ten provisioning a nejde to vytáhnout z google providera, takže se musí zadat, akorát jsem z toho zapomněl udělat env

Reply:
tak z toho udelej `locals` https://github.com/keboola/job-queue-daemon/blob/4d817a6b052118ed2c4cf4722c45d7ac1bd49326/provisioning/local/azure.tf#L2

jeslti vsichni budeme pozivat stejnej projekt (coz jsem tak pochopil z diskuze na Slacku), neni potreba lidi trapit nastavenim

Reply:
hmm, nj. vlastně, to jsem ještě nevědel, že to cpem do jednoho projektu :) tím líp